pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19649377?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19649377?buildTab=artifacts#/) on master @ [442e522f7fed1690f234bc794f8f5e6859710aca](https://github.com/cockroachdb/cockroach/commits/442e522f7fed1690f234bc794f8f5e6859710aca):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI3877201070
    tenant_status_test.go:145: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI3877201070
--- FAIL: TestTenantStatusAPI (15.60s)
=== RUN   TestTenantStatusAPI/tenant_contention_event
    tenant_status_test.go:1000: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/tenant_status_test.go:1000
        	            				pkg/ccl/serverccl/statusccl/tenant_status_test.go:90
        	Error:      	Should be true
        	Test:       	TestTenantStatusAPI/tenant_contention_event
        	Messages:   	expect to find contention event for table 114, but found events:<index_contention_events:<table_id:54 index_id:1 num_contention_events:11 cumulative_contention_time:<nanos:46963347 > events:<key:"\276\211\375\016\332\310\365xm\000\003\022legacy_payload\000\001\024\204\227\343\017\030\204\3145\204\037\210" txns:<txn_id:5e366a68-194e-420a-85b1-758e01054afa count:1 > > events:<key:"\276\211\375\016\332\310\365xm\000\003\022legacy_payload\000\001\024\204\227\343\017\030\204\321\212\376\237\210" txns:<txn_id:15f2a7c6-2581-4e4d-a92e-360810a39d1e count:1 > > events:<key:"\276\211\375\016\332\310\365\237X\000\003\022legacy_payload\000\001\024\204\227\343\017\027\204\372\206L\017\210" txns:<txn_id:816aab02-3e2c-4834-9d52-0a82dfa414b4 count:1 > > events:<key:"\276\211\375\016\332\310\365\305g\000\003\022legacy_payload\000\001\024\204\227\343\017\027\204\360Y\306?\210" txns:<txn_id:3065dd40-a1da-4045-85ca-e8d2a1192fbd count:1 > > events:<key:"\276\211\375\016\332\310\365\347\251\000\003\022legacy_payload\000\001\024\204\227\343\017\027\204\346p\315g\210" txns:<txn_id:f38b078d-1a81-49da-b023-5f7f4c38b3c4 count:1 > > events:<key:"\276\211\375\016\332\310\366\250\307\000\003\022legacy_payload\000\001\024\204\227\343\017\026\204\354\320\365o\210" txns:<txn_id:8dafe7cf-5964-4c79-87b1-e3f6f2647bcb count:1 > > events:<key:"\276\211\375\016\332\310\3672\270\000\003\022legacy_payload\000\001\024\204\227\343\017\025\204\367\316\005\367\210" txns:<txn_id:89378663-5dc5-4038-9742-fe26badff9d3 count:1 > > events:<key:"\276\211\375\016\332\310\3672\270\000\003\022legacy_payload\000\001\024\204\227\343\017\025\204\375L\267?\210" txns:<txn_id:481afbfa-136d-4baa-8c3c-d70512128fd9 count:1 > > events:<key:"\276\211\375\016\332\310\367N\327\200\003\022legacy_payload\000\001\024\204\227\343\017\025\204\357\222\rW\210" txns:<txn_id:bc082fb2-43cf-428a-a717-8cc1884af0e2 count:1 > > events:<key:"\276\211\375\016\332\310\367j\014\000\003\022legacy_payload\000\001\024\204\227\343\017\025\204\346K0\177\210" txns:<txn_id:7bdbdf1b-f020-473b-b62b-395630319ac4 count:1 > > events:<key:"\276\211\375\016\332\310\367\210\246\000\003\022legacy_payload\000\001\024\204\227\343\017\025\204\333R:\277\210" txns:<txn_id:46529a57-fa9c-4805-bea6-edfaef832d4b count:1 > > > index_contention_events:<table_id:15 index_id:1 num_contention_events:5 cumulative_contention_time:<nanos:17689741 > events:<key:"\227\211\375\016\332\310\365xm\000\003\212\211" txns:<txn_id:5e366a68-194e-420a-85b1-758e01054afa count:2 > > events:<key:"\227\211\375\016\332\310\365\347\251\000\003\212\211" txns:<txn_id:f38b078d-1a81-49da-b023-5f7f4c38b3c4 count:1 > > events:<key:"\227\211\375\016\332\310\370W\366\200\001\210" txns:<txn_id:6949c47e-5b8a-4576-9061-86a0fae943e3 count:1 > > events:<key:"\227\211\375\016\332\310\371\002\371\000\002\210" txns:<txn_id:701baf85-b6f5-42bc-8741-6442f972739a count:1 > > > index_contention_events:<table_id:15 index_id:4 num_contention_events:2 cumulative_contention_time:<nanos:16374554 > events:<key:"\227\214\022\001\001\200\020X\305!\347_H5\271\370+\215c\212\305?\000\001\022running\000\001\024\371h\034\360\343\371\033_\201\340\375\016\332\310\362)\242\000\001\210" txns:<txn_id:f0cc9351-e26b-4f4b-a8ae-92f4a4b772a3 count:1 > > events:<key:"\227\214\022\001\001\200\020X\305!\347_H5\271\370+\215c\212\305?\000\001\022running\000\001\024\371h\034\360\343\371\034D\005\340\375\016\332\310\362,\216\200\001\210" txns:<txn_id:e6199635-3a79-4138-b3b8-0cacd520a272 count:1 > > > index_contention_events:<table_id:11 index_id:3 num_contention_events:1 cumulative_contention_time:<nanos:1027390 > events:<key:"\223\213\022\200\000\001\361\211\022\001\001\200\020X\305!\347_H5\271\370+\215c\212\305?\000\001\210" txns:<txn_id:f33fe276-7576-4ded-9445-b995dc45b937 count:1 > > > > 
--- FAIL: TestTenantStatusAPI/tenant_contention_event (0.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19652461?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19652461?buildTab=artifacts#/) on master @ [995ee1d88581255c41b3d26c8968787cb2b8e158](https://github.com/cockroachdb/cockroach/commits/995ee1d88581255c41b3d26c8968787cb2b8e158):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI3439722759
    tenant_status_test.go:145: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI3439722759
--- FAIL: TestTenantStatusAPI (13.06s)
=== RUN   TestTenantStatusAPI/tenant_contention_event
    tenant_status_test.go:1000: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/tenant_status_test.go:1000
        	            				pkg/ccl/serverccl/statusccl/tenant_status_test.go:90
        	Error:      	Should be true
        	Test:       	TestTenantStatusAPI/tenant_contention_event
        	Messages:   	expect to find contention event for table 114, but found events:<index_contention_events:<table_id:54 index_id:1 num_contention_events:11 cumulative_contention_time:<nanos:38939180 > events:<key:"\276\211\375\016\332\367\245%\253\000\003\022legacy_payload\000\001\024\204\227\342\321\346\204\365<\260\237\210" txns:<txn_id:b753a371-c136-451a-bf30-9758bd7933fc count:1 > > events:<key:"\276\211\375\016\332\367\245%\253\000\003\022legacy_payload\000\001\024\204\227\342\321\346\204\373\302F\257\210" txns:<txn_id:cc3f8330-e1d2-435f-a98d-8817726ba204 count:1 > > events:<key:"\276\211\375\016\332\367\245?\335\200\003\022legacy_payload\000\001\024\204\227\342\321\346\204\357Q\235?\210" txns:<txn_id:eb0e4a54-5f89-46ff-a98f-357fd3715d87 count:1 > > events:<key:"\276\211\375\016\332\367\245Rw\200\003\022legacy_payload\000\001\024\204\227\342\321\346\204\350\232D\347\210" txns:<txn_id:d9ff98a2-8dbb-46fb-a4c5-4e7d45620819 count:1 > > events:<key:"\276\211\375\016\332\367\245hG\200\003\022legacy_payload\000\001\024\204\227\342\321\346\204\343<\337\017\210" txns:<txn_id:81ac0534-8496-4578-8b8e-4fac515b6022 count:1 > > events:<key:"\276\211\375\016\332\367\245\334\031\000\001\022legacy_payload\000\001\024\204\227\342\321\345\204\376\374K\237\210" txns:<txn_id:dafbeda9-9d2a-45a5-bbfe-ef2ce4d7efd3 count:1 > > events:<key:"\276\211\375\016\332\367\246>\r\200\001\022legacy_payload\000\001\024\204\227\342\321\345\204\334\367\005g\210" txns:<txn_id:bc2a4c98-b8d1-4c66-8f88-f87ee53f66ad count:1 > > events:<key:"\276\211\375\016\332\367\246>\r\200\001\022legacy_payload\000\001\024\204\227\342\321\345\204\340\371\035\027\210" txns:<txn_id:b14369f2-cbf0-4f59-9a75-f2223b3828bd count:1 > > events:<key:"\276\211\375\016\332\367\246S\226\000\001\022legacy_payload\000\001\024\204\227\342\321\345\204\326\216\003\277\210" txns:<txn_id:a75d502e-8f05-49ce-bf5b-cb9c66eeb8a6 count:1 > > events:<key:"\276\211\375\016\332\367\246f>\200\001\022legacy_payload\000\001\024\204\227\342\321\345\204\321\0038\317\210" txns:<txn_id:39c5cb66-99c6-460e-ac98-1833d4ef7218 count:1 > > events:<key:"\276\211\375\016\332\367\246y\264\200\001\022legacy_payload\000\001\024\204\227\342\321\345\204\312\302L\337\210" txns:<txn_id:780ecbc3-e667-45a2-b44c-821219a6a656 count:1 > > > index_contention_events:<table_id:15 index_id:1 num_contention_events:1 cumulative_contention_time:<nanos:6033451 > events:<key:"\227\211\375\016\332\367\247ky\200\002\210" txns:<txn_id:cbb11ff2-a2fa-4435-a08f-7142ddf01791 count:1 > > > > 
--- FAIL: TestTenantStatusAPI/tenant_contention_event (0.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://tanzanite.cluster.engflow.com/invocations/default/97a7d299-7979-4c0d-a1a9-6e1c9e263565?testReportRun=10&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3NlcnZlcmNjbC9zdGF0dXNjY2w6c3RhdHVzY2NsX3Rlc3Q=) on master @ [508919efdd82778cbf718f917e13ceca89020d39](https://github.com/cockroachdb/cockroach/commits/508919efdd82778cbf718f917e13ceca89020d39):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestTenantStatusAPI2912468131
    tenant_status_test.go:145: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: outputs.zip/logTestTenantStatusAPI2912468131
--- FAIL: TestTenantStatusAPI (14.56s)
=== RUN   TestTenantStatusAPI/tenant_contention_event
    tenant_status_test.go:1000: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/tenant_status_test.go:1000
        	            				pkg/ccl/serverccl/statusccl/tenant_status_test.go:90
        	Error:      	Should be true
        	Test:       	TestTenantStatusAPI/tenant_contention_event
        	Messages:   	expect to find contention event for table 114, but found events:<index_contention_events:<table_id:54 index_id:1 num_contention_events:11 cumulative_contention_time:<nanos:53395444 > events:<key:"\276\211\375\016\333D\211\327\271\200\003\022legacy_payload\000\001\024\204\227\342m\035\204\357\227\345o\210" txns:<txn_id:c4613cc9-a2a6-4327-9f74-616a564c62f0 count:1 > > events:<key:"\276\211\375\016\333D\211\327\271\200\003\022legacy_payload\000\001\024\204\227\342m\035\204\363s\004\217\210" txns:<txn_id:cda29393-a0f5-4312-b564-0f6de1799443 count:1 > > events:<key:"\276\211\375\016\333D\211\351I\200\003\022legacy_payload\000\001\024\204\227\342m\035\204\352f\017\357\210" txns:<txn_id:4b2fa3d4-0282-4bb3-8f29-c1113b7020a2 count:1 > > events:<key:"\276\211\375\016\333D\211\373\016\200\003\022legacy_payload\000\001\024\204\227\342m\035\204\344\317\376G\210" txns:<txn_id:2d5ca1f1-503b-4934-bee9-d10c62fda981 count:1 > > events:<key:"\276\211\375\016\333D\212\014\351\200\003\022legacy_payload\000\001\024\204\227\342m\035\204\337d\206\337\210" txns:<txn_id:31361b2a-4075-4ca7-95b6-971bd50b34e6 count:1 > > events:<key:"\276\211\375\016\333D\212\213}\000\001\022legacy_payload\000\001\024\204\227\342m\034\204\370,f\037\210" txns:<txn_id:b706a663-44df-4943-8ca9-d15a96b5e004 count:1 > > events:<key:"\276\211\375\016\333D\212\373\273\200\001\022legacy_payload\000\001\024\204\227\342m\034\204\321\335)\307\210" txns:<txn_id:0f894ba9-e576-43ec-a2ca-89b2fdfdcc73 count:1 > > events:<key:"\276\211\375\016\333D\212\373\273\200\001\022legacy_payload\000\001\024\204\227\342m\034\204\325\352\322g\210" txns:<txn_id:02609c53-b7d0-4928-b1c1-2fad232db630 count:1 > > events:<key:"\276\211\375\016\333D\213\017*\200\001\022legacy_payload\000\001\024\204\227\342m\034\204\313\277\234\207\210" txns:<txn_id:44288410-2a71-49f0-9943-a17496233b8b count:1 > > events:<key:"\276\211\375\016\333D\213#\007\200\001\022legacy_payload\000\001\024\204\227\342m\034\204\306A\n\177\210" txns:<txn_id:a2bda84d-25a9-4049-8747-f56875d0b8f5 count:1 > > events:<key:"\276\211\375\016\333D\2133\250\000\001\022legacy_payload\000\001\024\204\227\342m\033\204\375\017)o\210" txns:<txn_id:76a5a2a5-c659-4089-8de0-52e4cef2a169 count:1 > > > index_contention_events:<table_id:15 index_id:1 num_contention_events:7 cumulative_contention_time:<nanos:12657269 > events:<key:"\227\211\375\016\333D\206\250T\200\001\212\211" txns:<txn_id:da594412-f088-4ec0-8863-cd45ecacfd51 count:1 > > events:<key:"\227\211\375\016\333D\211\373\016\200\003\212\211" txns:<txn_id:2d5ca1f1-503b-4934-bee9-d10c62fda981 count:1 > > events:<key:"\227\211\375\016\333D\212\373\273\200\001\210" txns:<txn_id:02609c53-b7d0-4928-b1c1-2fad232db630 count:1 > > events:<key:"\227\211\375\016\333D\212\373\273\200\001\212\211" txns:<txn_id:0f894ba9-e576-43ec-a2ca-89b2fdfdcc73 count:2 > > events:<key:"\227\211\375\016\333D\213\017*\200\001\212\211" txns:<txn_id:44288410-2a71-49f0-9943-a17496233b8b count:1 > > events:<key:"\227\211\375\016\333D\214\257\340\000\002\210" txns:<txn_id:a5611bc6-2d34-4f06-bb70-3110ee5ec950 count:1 > > > > 
--- FAIL: TestTenantStatusAPI/tenant_contention_event (0.14s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=10</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19658156?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19658156?buildTab=artifacts#/) on master @ [a717f19730e2570a01fdaf57511a9645c73a7aea](https://github.com/cockroachdb/cockroach/commits/a717f19730e2570a01fdaf57511a9645c73a7aea):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI1915904603
    tenant_status_test.go:145: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI1915904603
--- FAIL: TestTenantStatusAPI (13.00s)
=== RUN   TestTenantStatusAPI/tenant_contention_event
    tenant_status_test.go:1000: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/tenant_status_test.go:1000
        	            				pkg/ccl/serverccl/statusccl/tenant_status_test.go:90
        	Error:      	Should be true
        	Test:       	TestTenantStatusAPI/tenant_contention_event
        	Messages:   	expect to find contention event for table 114, but found events:<index_contention_events:<table_id:54 index_id:1 num_contention_events:11 cumulative_contention_time:<nanos:48212585 > events:<key:"\276\211\375\016\333\307\232R\331\000\003\022legacy_payload\000\001\024\204\227\341\301S\204\3654\251\357\210" txns:<txn_id:ea8a6a8a-d0c2-42eb-b4d8-ce5b3340dc4e count:1 > > events:<key:"\276\211\375\016\333\307\232R\331\000\003\022legacy_payload\000\001\024\204\227\341\301S\204\371^\366\227\210" txns:<txn_id:1041ffb7-d6ea-46bd-b03d-9fdbc200e370 count:1 > > events:<key:"\276\211\375\016\333\307\232f\026\200\003\022legacy_payload\000\001\024\204\227\341\301S\204\357\261\227\217\210" txns:<txn_id:f4fe9903-e726-40bb-8bad-483e2aef21ee count:1 > > events:<key:"\276\211\375\016\333\307\232x\366\200\003\022legacy_payload\000\001\024\204\227\341\301S\204\351\3410O\210" txns:<txn_id:e80276b8-db27-489b-aff9-7ad5987e2a5f count:1 > > events:<key:"\276\211\375\016\333\307\232\214\310\200\003\022legacy_payload\000\001\024\204\227\341\301S\204\343\375L\307\210" txns:<txn_id:63e625cb-0fe5-44c0-8502-8e11f143d545 count:1 > > events:<key:"\276\211\375\016\333\307\233\002O\200\001\022legacy_payload\000\001\024\204\227\341\301R\205d\344\367\210" txns:<txn_id:8c7a5ecc-2d9a-4b20-890c-2e7b120e6d14 count:1 > > events:<key:"\276\211\375\016\333\307\233e\230\200\001\022legacy_payload\000\001\024\204\227\341\301R\204\334\301\215\347\210" txns:<txn_id:6497a4d3-80ef-46ca-af05-ade898e52abf count:1 > > events:<key:"\276\211\375\016\333\307\233e\230\200\001\022legacy_payload\000\001\024\204\227\341\301R\204\341\265)\267\210" txns:<txn_id:972ce2a1-d975-467d-a43a-14ede34c7237 count:1 > > events:<key:"\276\211\375\016\333\307\233z\377\000\001\022legacy_payload\000\001\024\204\227\341\301R\204\327\004l?\210" txns:<txn_id:9d87c39d-809a-4f17-b90c-c705df4cd2b8 count:1 > > events:<key:"\276\211\375\016\333\307\233\213\375\000\001\022legacy_payload\000\001\024\204\227\341\301R\204\321\341\216\307\210" txns:<txn_id:2b670ee6-f34c-4b9b-9d57-53ffb5dee2ed count:1 > > events:<key:"\276\211\375\016\333\307\233\235\007\200\001\022legacy_payload\000\001\024\204\227\341\301R\204\314\\\332o\210" txns:<txn_id:80b1b848-06c7-4da8-a4c2-8f48a2cb3dfe count:1 > > > index_contention_events:<table_id:15 index_id:1 num_contention_events:1 cumulative_contention_time:<nanos:5095880 > events:<key:"\227\211\375\016\333\307\234\236\006\200\002\210" txns:<txn_id:bfd89253-8b6f-41e0-bda8-c664432c3aa0 count:1 > > > > 
--- FAIL: TestTenantStatusAPI/tenant_contention_event (0.13s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19659109?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19659109?buildTab=artifacts#/) on master @ [b1b1b4f1d39ebf029f3d25042000a6691d6273d6](https://github.com/cockroachdb/cockroach/commits/b1b1b4f1d39ebf029f3d25042000a6691d6273d6):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI1693887549
    tenant_status_test.go:145: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI1693887549
--- FAIL: TestTenantStatusAPI (12.57s)
=== RUN   TestTenantStatusAPI/tenant_contention_event
    tenant_status_test.go:1000: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/tenant_status_test.go:1000
        	            				pkg/ccl/serverccl/statusccl/tenant_status_test.go:90
        	Error:      	Should be true
        	Test:       	TestTenantStatusAPI/tenant_contention_event
        	Messages:   	expect to find contention event for table 114, but found events:<index_contention_events:<table_id:54 index_id:1 num_contention_events:12 cumulative_contention_time:<nanos:52393737 > events:<key:"\276\211\375\016\333\337\272\245P\000\003\022legacy_payload\000\001\024\204\227\341\241\264\204\342\345{\367\210" txns:<txn_id:4de5abb8-8515-4b17-9832-5a991c71cca7 count:1 > > events:<key:"\276\211\375\016\333\337\273\035T\000\003\022legacy_payload\000\001\024\204\227\341\241\263\204\360\307\302\177\210" txns:<txn_id:2f25b4e7-c1a3-4432-a108-8c86144a1399 count:1 > > events:<key:"\276\211\375\016\333\337\273\035T\000\003\022legacy_payload\000\001\024\204\227\341\241\263\204\367\242\250g\210" txns:<txn_id:f415620f-c057-4203-9c9a-cd3ca1affdde count:1 > > events:<key:"\276\211\375\016\333\337\2739\004\200\003\022legacy_payload\000\001\024\204\227\341\241\263\204\350]0/\210" txns:<txn_id:a2e1061e-d214-4c7c-815c-6781d98721c1 count:1 > > events:<key:"\276\211\375\016\333\337\273Q\373\200\003\022legacy_payload\000\001\024\204\227\341\241\263\204\341\220_\277\210" txns:<txn_id:b568988d-8a71-4af8-a56f-76d7e23f51cf count:1 > > events:<key:"\276\211\375\016\333\337\273i\337\000\003\022legacy_payload\000\001\024\204\227\341\241\263\204\331\377=\177\210" txns:<txn_id:38a8eab5-d935-4dd0-91da-fae67658c55f count:1 > > events:<key:"\276\211\375\016\333\337\274\n\360\000\003\022legacy_payload\000\001\024\204\227\341\241\262\204\351\231F'\210" txns:<txn_id:3cab149a-94ec-431e-b0ec-5165a79abd00 count:1 > > events:<key:"\276\211\375\016\333\337\274\203C\000\003\022legacy_payload\000\001\024\204\227\341\241\261\204\374\004\227?\210" txns:<txn_id:e18ca67d-b2ed-4069-856c-bd08b7375f77 count:1 > > events:<key:"\276\211\375\016\333\337\274\203C\000\003\022legacy_payload\000\001\024\204\227\341\241\262\204\305M\367\357\210" txns:<txn_id:00976318-28b3-45a5-afd7-183b263e8cd3 count:1 > > events:<key:"\276\211\375\016\333\337\274\236\260\200\003\022legacy_payload\000\001\024\204\227\341\241\261\204\360\372\212\177\210" txns:<txn_id:1746cd58-cabc-4533-b94d-d7ba86d23e5b count:1 > > events:<key:"\276\211\375\016\333\337\274\274\367\200\003\022legacy_payload\000\001\024\204\227\341\241\261\204\351\306\026\317\210" txns:<txn_id:a20b9f81-5284-4388-a84f-d6f7bf58f30f count:1 > > events:<key:"\276\211\375\016\333\337\274\327\202\000\003\022legacy_payload\000\001\024\204\227\341\241\261\204\342\207+o\210" txns:<txn_id:252134e5-6e5e-493c-b554-8426d4be271f count:1 > > > index_contention_events:<table_id:15 index_id:1 num_contention_events:4 cumulative_contention_time:<nanos:17504929 > events:<key:"\227\211\375\016\333\337\274\n\360\000\003\210" txns:<txn_id:3cab149a-94ec-431e-b0ec-5165a79abd00 count:1 > > events:<key:"\227\211\375\016\333\337\274\203C\000\003\210" txns:<txn_id:00976318-28b3-45a5-afd7-183b263e8cd3 count:1 > > events:<key:"\227\211\375\016\333\337\275\211e\000\001\210" txns:<txn_id:37d3c7ee-40f4-43b2-bdd5-8dd034249601 count:1 > > events:<key:"\227\211\375\016\333\337\276\004\263\000\002\210" txns:<txn_id:21492661-9c40-43cf-8fca-252939f0ed72 count:1 > > > index_contention_events:<table_id:15 index_id:4 num_contention_events:2 cumulative_contention_time:<nanos:17041570 > events:<key:"\227\214\022\001\001\200\2558\331?\260\242NT\243\201\303\335\356\377\035\200\000\001\022running\000\001\024\371h\036^H\371/l\336\200\375\016\333\337\270\211\347\000\001\210" txns:<txn_id:14fd8224-c58a-439c-8be9-e586c7cff095 count:1 > > events:<key:"\227\214\022\001\001\200\2558\331?\260\242NT\243\201\303\335\356\377\035\200\000\001\022running\000\001\024\371h\036^H\3710\2422\000\375\016\333\337\270\215\334\200\001\210" txns:<txn_id:9a273f29-8a5a-41d6-9d7c-922c273aac82 count:1 > > > index_contention_events:<table_id:11 index_id:3 num_contention_events:1 cumulative_contention_time:<nanos:426095 > events:<key:"\223\213\022\200\000\001\360\211\022\001\001\200\250yA=csJ\373\267\210\017\000\377\375\220\251r\000\001\210" txns:<txn_id:8bf0b64f-2301-4644-9c87-3d1fa3c9c2cb count:1 > > > > 
--- FAIL: TestTenantStatusAPI/tenant_contention_event (0.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/serverccl/statusccl/tenant_status_test.go
+++ b/pkg/ccl/serverccl/statusccl/tenant_status_test.go
@@ -51,7 +51,6 @@ func TestTenantStatusAPI(t *testing.T) {
 	defer s.Close(t)
 	defer s.SetupSingleFileLogging()()
 
-	skip.WithIssue(t, 146387)
 	// The liveness session might expire before the stress race can finish.
 	skip.UnderRace(t, "expensive tests")
 
@@ -73,6 +72,10 @@ func TestTenantStatusAPI(t *testing.T) {
 	tdb.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.target_duration = '10ms'")
 	tdb.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '10 ms'")
 	tdb.Exec(t, "SET CLUSTER SETTING kv.rangefeed.closed_timestamp_refresh_interval = '10 ms'")
+	// If we happen to enable buffered writes metamorphically, we must have the
+	// split lock reliability enabled (which can be tweaked metamorphically too,
+	// #146387).
+	tdb.Exec(t, "SET CLUSTER SETTING kv.lock_table.unreplicated_lock_reliability.split.enabled = true")
 
 	t.Run("reset_sql_stats", func(t *testing.T) {
 		skip.UnderDeadlockWithIssue(t, 99559)
@@ -884,7 +887,6 @@ WHERE tablename = 'test' AND indexname = $1`
 		requireAfter(t, &resp.Statistics[0].Statistics.Stats.LastRead, &timePreRead)
 		indexName := resp.Statistics[0].IndexName
 		createStmt := cluster.TenantConn(0).QueryStr(t, getCreateStmtQuery, indexName)[0][0]
-		print(createStmt)
 		require.Equal(t, resp.Statistics[0].CreateStatement, createStmt)
 		requireBetween(t, timePreCreate, resp.Statistics[0].CreatedAt, timePreRead)
 	})

--- a/pkg/server/application_api/contention_test.go
+++ b/pkg/server/application_api/contention_test.go
@@ -24,7 +24,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sessiondata"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -37,12 +36,15 @@ func TestStatusAPIContentionEvents(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.WithIssue(t, 146412)
-
 	ctx := context.Background()
 	testCluster := serverutils.StartCluster(t, 3, base.TestClusterArgs{})
-
 	defer testCluster.Stopper().Stop(ctx)
+
+	hostRunner := sqlutils.MakeSQLRunner(testCluster.SystemLayer(0).SQLConn(t))
+	// If we happen to enable buffered writes metamorphically, we must have the
+	// split lock reliability enabled (which can be tweaked metamorphically too,
+	// #146412).
+	hostRunner.Exec(t, "SET CLUSTER SETTING kv.lock_table.unreplicated_lock_reliability.split.enabled = true")
 
 	s0 := testCluster.Server(0).ApplicationLayer()
 	s1 := testCluster.Server(1).ApplicationLayer()

